duckduckgo/iOS
# Fire Window on macOS

Task/Issue URL: https://app.asana.com/0/72649045549333/1203956617610891/f
Tech Design URL:  https://app.asana.com/0/481882893211075/1204384679803788/f
CC:

**Description**:
Making Fire Window available to public on macOS



**Steps to test this PR**:
1. Make sure iOS autofill feature works as expected: Users can save and retrieve credentials



**OS Testing**:



---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
